app/Plugin/EnvSource/Lib/enum.php

```diff
@@ -31,3 +31,8 @@ class EnvSourceDuplicateModeEnum
   const LoginIdentifier = "LI";
   const SORIdentifier   = "SI";
 }
+
+class AuthProviderEnum
+{
+  const Shibboleth   = ";";
```

Comment:
This is going to make SQL query construction really annoying :) The enum const should be something like "SH" (for "Shibboleth").

We should also define "Other" (code "O" for consistency") so if any deployer is using something else they don't get confused. We could proactively define "SimpleSamlPHP" ("SS"?) as well even though we're not doing anything specific for it.

Reply:
Done